api/files/views.py

```diff
@@ -311,6 +315,60 @@ def get_object(self):
         return self.get_file()
 
 
+class FileList(JSONAPIBaseView, generics.ListAPIView, NodeMixin, ListFilterMixin, WaterButlerMixin, bulk_views.BulkUpdateJSONAPIView):
+    """Special case for getting lists from files on a node stored with OSFStorage.
```

Comment:
Just to document, checked with @brianjgeiger and he said to remove docstrings as we don't document bulk API routes just yet